sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/ReadingOrder.java

```diff
@@ -3,15 +3,27 @@
 
 package com.azure.ai.formrecognizer.models;
 
-/** Defines values for ReadingOrder. */
+/**
+ * Defines values for ReadingOrder.
+ */
 public enum ReadingOrder {
-    /** Enum value basic. */
+
+    /**
+     * Enum value basic.
+     * Set it to basic for the lines to be sorted top to bottom, left to right, although in certain cases
+     * proximity is treated with higher priority.
+     */
     BASIC("basic"),
 
-    /** Enum value natural. */
+    /**
+     * Enum value natural.
+     * Set it to "natural" value for a more human-friendly reading order output.
```

Comment:
I want to make sure we use the correct description for this field across languages. Maybe during scrum we can agree on something

Reply:
Agree with this! 
For Python I described it like this:
```
            Reading order algorithm to sort the text lines returned. Supported
            reading orders include: basic (default), natural. Set 'basic' to sort lines left to right and top
            to bottom, although in some cases proximity is treated with higher priority. Set 'natural' to sort
            lines by those which are more closely related to each other.
```

Reply:
@witemple-msft Do you want to paste the docs used in JS as well and we all can just vote here and decide on this?